### PR TITLE
[UI/UX:Calendar] Fix calendar Note cursor

### PR DESCRIPTION
**Pull request by sadam-04**
### Please check if the PR fulfills these requirements:

* [ ] Tests for the changes have been added/updated (if possible)
* [ ] Documentation has been updated/added if relevant
* [ ] Screenshots are attached to Github PR if visual/UI changes were made

Fixes issue [#10171](https://github.com/Submitty/Submitty/issues/10171) 
This PR fixes an almost identical bug as was fixed in PR [#10125](https://github.com/Submitty/Submitty/pull/10125), except this fixes it for notes, instead of announcements in the calendar.

### What is the current behavior?
As the cursor is moved over the edge of a Note entry in the calendar view, the cursor changes to a pointer for a split second, before switching back to the default arrow cursor.

### What is the new behavior?
The cursor stays as an arrow, and doesn't change when dragged over the edge of Notes.

### Other information?
This is not a breaking change.
Tested from student, grader, and ta logins.

### CODE REVIEW

**Review by michaelkrar — APPROVED** [review]
Had difficulty replicating the issue at first. Looks like in a firefox browser(on ubuntu LTS) the bug is either too fast to see or doesn't occur, while in chrome it did visibly occur. (not very relevant but worth pointing out)

When testing the fix, the behavior in firefox appears to be identical to that without the fix, but when I tested it in chrome the behavior was fixed. 

Tested with student, ta, grader on chrome and firefox and the behavior was correct for all.
Also tested instructor specific functionalities in the calendar view (making new notes, announcements) and all worked as intended.
The fix also does not appear to be a breaking change.

**Review by arougey — APPROVED** [review]
Tested it in Firefox and could not reproduce the error.  Tested on student, instructor and ta logins. Based on the changed files, it looks like an effective fix for other browsers

File: site/public/css/calendar.css [hunk 1]
```diff
@@ -202,13 +202,14 @@ a.cal-gradeable-status-text {
...
     box-shadow: none;
+    cursor: default;
 }
 
 a.cal-gradeable-status-text:hover {
     color: var(--text-black);
     padding: 0;
-    cursor: default;
     border: none;
+    cursor: default;
```
Comment (cjreed121) [inline site/public/css/calendar.css:212]: Revert this change

File: site/public/css/calendar.css [hunk 2]
```diff
@@ -228,11 +229,11 @@ a.cal-gradeable-status-ann {
 }
 
 a.cal-gradeable-status-ann:hover {
-    cursor: default;
     padding: 0;
     font-style: italic;
     border-style: solid;
     border-width: 3px!important;
+    cursor: default;
```
Comment (cjreed121) [inline site/public/css/calendar.css:236]: Revert this change

**Review by cjreed121 — CHANGES_REQUESTED** [review]
This does fix the issue and is good functionality wise but please address the following requested changes.

**Comment by williamlin6803** [comment]
Tested it on my machine and it worked as normal.

**Review by ryvaru — APPROVED** [review]
Tested this code on my machine and everything works as intended, the cursor no longer changes when dragged over the edge of a note. Code looks logical and fixes the issue. Approved.

**Comment by AavanVadiwala** [comment]
Test as an instructor, student, and ta. Approved. When the cursor is dragged along the edge of the border of a note, the cursor remains an arrow unless dragged into the note, which then the cursor is switched to a pointer.

**Review by AavanVadiwala — APPROVED** [review]
Test as an instructor, student, and ta. Approved. When the cursor is dragged along the edge of the border of a note, the cursor remains an arrow unless dragged into the note, which then the cursor is switched to a pointer.

**Review by cjreed121 — APPROVED** [review]

**Pull request merged**